packages/loot-core/src/server/spreadsheet/new/nodes.ts

```diff
@@ -1,14 +1,12 @@
 class Node {
-  constructor(lineno, colno, fieldNames, ...fields) {
+  colno;
```

Comment:
Had to refactor these classes more than expected as TS + prototypal inheritance = pain with dynamic fields. So we no longer assign those fields to Node but to the extended class (which might make more sense).